samples/streaming_sample/sample_stream.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	openapi "github.com/deepfence/golang_deepfence_sdk/client"
+	oahttp "github.com/deepfence/golang_deepfence_sdk/utils/http"
+	"github.com/rs/zerolog/log"
+)
+
+// Please set the below host, port and token to appropriate values
+// as per the setup
+var (
+	console_host      = "134.211.211.211" //IP of the host running management console
```

Comment:
Can you add these as an argument?
https://github.com/deepfence/deepfence_runtime_api/blob/master/scripts/streaming_api/streaming_api.go#L17